.github/workflows/audit.yml

```
@@ -7,9 +7,11 @@ on:
     branches: [main]
   schedule:
     - cron: "40 10 * * *"
-
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 jobs:
   audit:
-    uses: inrupt/javascript-style-configs/.github/workflows/reusable-audit.yml@v1.1.0
+    uses: inrupt/typescript-sdk-tools/.github/workflows/reusable-lint.yml@v1.4.2
```

Copy and paste error: 
```suggestion
    uses: inrupt/typescript-sdk-tools/.github/workflows/reusable-audit.yml@v1.4.2
```

---

Oh good catch thanks